src/github.com/getlantern/flashlight/client/fronted.go

```diff
@@ -76,7 +75,7 @@ func (s *FrontedServerInfo) dialer(masqueradeSets map[string][]*fronted.Masquera
 		OnDialStats:        s.onDialStats,
 		Masquerades:        masqueradeSets[s.MasqueradeSet],
 		MaxMasquerades:     s.MaxMasquerades,
-		RootCAs:            globals.TrustedCAs,
+		RootCAs:            getCertPool(),
```

Comment:
Guess we can eliminate `RootCAs` field here, and let `fronted/dialer.go#L336` to use the one passed to `fronted.Configure`?